src/reporters/web-test-runner.js

```diff
@@ -0,0 +1,138 @@
+import { determineReportPath, getReportConfiguration, getOperatingSystem, writeReport, makeLocation, getReportOptions } from './helpers.cjs';
+import { randomUUID } from 'crypto';
+
+const collectTests = (reportConfiguration, session, prefix, tests) => {
+	const location = makeLocation(session.testFile);
+	const flattened = [];
+
+	for (const test of tests) {
+		let status;
+
+		if (test.skipped) {
+			status = 'skipped';
+		} else if (test.passed) {
+			status = 'passed';
+		} else {
+			status = 'failed';
+		}
+
+		const { type, tool, experience } = getReportOptions(reportConfiguration, location);
+		const duration = test.duration ?? 0;
+
+		flattened.push({
+			name: `${prefix}${test.name}`,
+			duration: duration,
+			totalDuration: duration,
+			status,
+			location,
+			type,
+			tool,
+			experience,
+			retries: 0,
+			started: (new Date()).toISOString()
```

Comment:
Is an individual test start time useful? It would mean any time filtering that just uses the `details` table would have to join with the summary table to get a time value. Time stream also requires this but we could always just write it as the report summary start time at record submission.

Reply:
I think the overall run time for a test is more valuable than just the start time